javascript/modules/client/test/integration/client-addressList.test.ts

```diff
@@ -340,28 +338,28 @@ describe("Client", () => {
         ]
       };
 
-      const initAction = ClientAddressList.encoding.installEntry(withdrawParams);
+      const installEntry = ClientAddressList.encoding.installEntry(withdrawParams);
```

Comment:
Triple checking name consistency (not for the test, but from the SDK user perspective)

Reply:
Not sure what are you referring to

Reply:
Cosmetic, we can ignore this as part of the mock